world_state start_state
motion_plan[] plan
